openpype/settings/entities/schemas/projects_schema/schemas/schema_maya_publish.json

```diff
@@ -309,6 +309,24 @@
             "type": "schema",
             "name": "schema_maya_capture"
         },
+        {
```

Comment:
There is `Submit maya job to deadline` in `Deadline` settings to separate deadline.

![image](https://user-images.githubusercontent.com/43494761/124882231-74c21480-dfd0-11eb-93f7-e97e5da78a9d.png)